openassessment/xblock/static/js/src/oa_server.js

```diff
@@ -531,6 +531,26 @@ if (typeof OpenAssessment.Server === "undefined" || !OpenAssessment.Server) {
             }).promise();
         },
 
+        /**
+         * Sends request to server to remove specific uploaded file.
+         */
+        removeUploadedFile: function(filenum) {
+            var url = this.url('remove_uploaded_file');
+            return $.Deferred(function(defer) {
+                $.ajax({
+                    type: "POST",
+                    url: url,
+                    data: JSON.stringify({'filenum': filenum}),
```

Comment:
nit: what about calling this file fileId, fileIndex or fileNumber (spelled out)

Reply:
filenum is used other places in this file so I just wanted to be consistent

Reply:
as well as in other files in this general workflow